tests/dragonfly/acl_family_test.py

```diff
@@ -438,3 +438,32 @@ async def test_set_acl_file(async_client: aioredis.Redis, tmp_dir):
 
     result = await async_client.execute_command("AUTH roy mypass")
     assert result == "OK"
+
+
+@pytest.mark.asyncio
```

Comment:
This test fails.

There are two issues: 

1. `resize` above is not good, because it could be the case that the log is not full, and resize will introduce `default` constructed elements and increase the log size. See the above suggestion to fix this.
2. This test needs some small changes. Basically, the problem is, that each `proactor` has its own `log`. So if you run this test on a machine with 8 proactors, then the total elements you gonna get back is (8 * acllog_max_len). What I suggest here is, to create a df instance instead of using `async_client` with `proactor_threads=1`. That way, the assertions below should work outside of the box :)

Reply:
Done, Thanks for your comments.